src/App/Components/ButtonComponent/ButtonComponent.ts

```
@@ -0,0 +1,13 @@
+import IComponent from '../IComponent/IComponent';
+
+import ButtonComponentTemplate from './ButtonComponent.hbs';
+import ButtonComponentData from "./ButtonComponentData";

```

Такие импорты можно заменить на import type, чтобы лишнее из модулей не попадало в рантайм бандл https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html
Сейчас это ни на что не повлияет, но если ты в этот файлик добавишь какой-то код и не будет его тут использовать, он всё равно попадёт в бандл, потому что из этого файла есть импорт
Если делать `import type`, то модуль не попадёт в бандл, а будет использован лишь для проверки типов на моменте компиляции
Можно включить опцию [importsNotUsedAsValues](https://www.typescriptlang.org/tsconfig#importsNotUsedAsValues) = `error`, чтобы typescript подсвечивал сам такие места

---

Поставил и получил 136 ошибок))) Пришлось минут 8 добавлять type) Но почему-то не получилось запустить с этим свойством build, так как оно было убрано в ts 5.5, получил вот такую ошибку:
ERROR
      TS5101: Option 'importsNotUsedAsValues' is deprecated and will stop functioning in TypeScript 5.5. Specify compilerOption '"ignoreDeprecations": "5.0"' to silence this error.    
  Use 'verbatimModuleSyntax' instead.
  Поставил 'verbatimModuleSyntax' вместо 'importsNotUsedAsValues' и получил ещё 42 ошибки))
  Всё пофиксил и теперь всё работает